src/components/ui/form-textarea/FormTextarea.vue

```diff
@@ -1,59 +1,88 @@
 <script setup lang="ts">
 import { computed } from 'vue';
+import { useMaxLength } from '../form-wrapper/composables/useMaxLength';
 import FormWrapper from '../form-wrapper/FormWrapper.vue';
-import type { FormTextareaProps } from './types';
-
-const emit = defineEmits<{
-  (e: 'update:modelValue', val: string): void;
-  (e: 'update', val: string): void;
-}>();
+import type { FormTextareaEmits, FormTextareaProps } from './types';
 
+const model = defineModel<string>();
 const props = withDefaults(defineProps<FormTextareaProps>(), {
   rows: 4,
   state: undefined,
+  loading: false,
+  disabled: false,
+  autofocus: false,
+  float: false,
+  allowExceedMaxLength: false,
+  textsCounter: () => ({
+    counterInitialLimit: 'Você pode digitar até {{amount}} {{element}}',
+    counterRemaining: 'Você tem {{amount}} {{element}} {{remaining}}',
+    counterExceeded: 'Você atingiu o limite de {{element}}',
+    counterReachedLimit: 'Você excedeu o limite em {{amount}} {{element}}',
+    wordRemaining: {
+      singular: 'restante',
+      plural: 'restantes',
+    },
+    wordElement: {
+      singular: 'caractere',
+      plural: 'caracteres',
+    },
+  }),
```

Comment:
Não seria interessante transformar essa parte num composable para poder reusar onde for implementado o contador?